src/components/PrintLibraryCardButton/PrintLibraryCardButton.js

```diff
@@ -1,18 +1,74 @@
-import { FormattedMessage } from 'react-intl';
+import { useMemo, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { cloneDeep } from 'lodash';
+import moment from 'moment';
 
-import { Button, Icon } from '@folio/stripes/components';
+import {
+  Button,
+  Icon,
+  exportToCsv,
+} from '@folio/stripes/components';
+
+const PrintLibraryCardButton = ({ user, patronGroup }) => {
+  const intl = useIntl();
+  const onlyFields = useMemo(() => [
```

Comment:
`useMemo` is not required here, because it cost more then potential optimization. `onlyFields` can be moved outside of component because it's constant, and it will be stable between re-renders.